src/index.html

```diff
@@ -17,6 +21,94 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="header-container">
+        <div class="logo-container">
+          <a href="#">
+            <img
+              src="./images/logo.png"
+              alt="MOYO Logo"
+              class="logo"
+            />
```

Comment:
The 'href' attribute in the 'a' tag should not have a space (it should start with the '#' symbol if it's meant to be a placeholder).